Mathlib/Algebra/Free.lean

```diff
@@ -230,8 +230,7 @@ theorem traverse_mul' :
 @[to_additive (attr := simp)]
 theorem traverse_eq (x) : FreeMagma.traverse F x = traverse F x := rfl
 
--- Porting note (#10675): dsimp can not prove this
-@[to_additive (attr := simp, nolint simpNF)]
+@[to_additive]
```

Comment:
What's the justification of removing the porting note and `simp` attribute here?

Reply:
Ah, I presume it's similar to the `FreeSemigroup` version below!